diff --git a/cuda/preconditioner/jacobi_kernels.cu b/cuda/preconditioner/jacobi_kernels.cu
index d594522e..1db6390c 100644
--- a/cuda/preconditioner/jacobi_kernels.cu
+++ b/cuda/preconditioner/jacobi_kernels.cu
@@ -58,7 +58,7 @@ namespace cuda {
  * A compile-time list of block sizes for which dedicated generate and apply
  * kernels should be compiled.
  */
-using compiled_kernels = syn::compile_int_list<1, 13, 16, 32>;
+using compiled_kernels = syn::compile_int_list<>;
 
 
 namespace {
